Source/Scene/GroundPrimitive.js

```diff
@@ -39,28 +42,27 @@ define([
         OrientedBoundingBox,
         Rectangle,
         Resource,
+        DrawCommand,
         Pass,
         when,
         ClassificationPrimitive,
         ClassificationType,
-        SceneMode) {
+        PerInstanceColorAppearance,
+        SceneMode,
+        ShadowVolumeAppearance) {
     'use strict';
 
-    var GroundPrimitiveUniformMap = {
```

Comment:
I wasn't sure if there would be problems with material uniforms, but on closer inspection it looks like `combine` spits out a brand new object so that shouldn't be a problem?